src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -75,6 +81,7 @@ public enum LiftPositionStartRelative {
   private LiftPosition latestPosition = LiftPosition.STARTING;
   private LiftPosition desiredPosition = LiftPosition.STARTING;
   private boolean desiredGrabberClosed = true;
+  private ScoringLocationUtil scoreLoc = new ScoringLocationUtil();
```

Comment:
A new `ScoringLocationUtil` here won't get the updates from the driver controllers. We should pass in the one in robotContainer to the Lift constructor and just save it to this member variable.